frontend/public/components/modals/delete-modal.jsx

```diff
@@ -43,7 +43,7 @@ class DeleteModal extends PromiseComponent {
 
   render() {
     const {kind, resource} = this.props;
-    return <form onSubmit={this._submit} name="form">
+    return <form onSubmit={this._submit} name="form" className="modal-content">
       <ModalTitle>Delete {kind.label}</ModalTitle>
       <ModalBody className="modal-body co-delete-modal">
```

Comment:
Note the passing of `className` here overrides the default classNames in modal.tsx.  As a result, this modal doesn't get the height adjustments.  Is that intentional?